app/views/warehouse_reports/health/expiring_items/_chas.haml

```diff
@@ -0,0 +1,30 @@
+- if @report.expiring_cha_forms.exists?
+  .expiring-cha-forms.mr-6
+    %h2 Expiring Comprehensive Health Assessments
+    %p The following patients have CHAs that will expire in the next month.
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Patient
+          %th Expiration Date
+      %tbody
+        - @report.expiring_cha_forms.each do |patient|
+          %tr
+            %td= link_to patient.name, client_health_patient_index_path(patient)
+            %td= patient.recent_cha_form.expires_on || 'Incomplete'
+
+- if @report.expired_cha_forms.exists?
+  .expired-cha-forms
+    %h2 Expired Comprehensive Health Assessments
+    %p The following patients have CHAs that have expired.
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Patient
+          %th Expiration Date
+      %tbody
+        - @report.expired_cha_forms.each do |patient|
+          %tr
+            %td= link_to patient.name, client_health_patient_index_path(patient)
+            %td= patient.recent_cha_form.expires_on || 'Incomplete'
+
```

Comment:
Should there be a 'No expired or expiring CHAs' etc?

Reply:
I simply didn't show them if there weren't any.  It seemed cleaner, but maybe.